src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Kindly remove these `eslint-disable` comments as adding them defeats the purpose of linters. Fix the errors instead.

  The first one seems to be unnecessary as when I removed it eslint throws no errors. For the second one you need to add a state to the component.

  ```
  export default class Calculator extends Component {
    constructor(props) {
      super(props);
  
      this.state = {};
    }
  ...
  ```